packages/nc-gui/components/cell/SingleSelect/utils.ts

```
@@ -0,0 +1,71 @@
+        op.title = op.title.replace(/^'/, '').replace(/'$/, '')
+      }
+
+      let order = 1
+      const limitOptionsById =
+        ((parseProp(column.value.meta)?.limitOptions || []).reduce(
+          (o: Record<string, FormFieldsLimitOptionsType>, f: FormFieldsLimitOptionsType) => {
+            if (order < (f?.order ?? 0)) {
+              order = f.order
+            }
+            return {
+              ...o,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)